### PR TITLE
Add kind type to root span to fix the empty parentID problem

### PR DESCRIPTION
**Pull request by JohnWu20**
**Description:**

Issue:  The xray export couldn't tell the root span because the xray receiver didn't set the span kind type for it.

**Testing:** 

Did the local test, and checked the multi-span data in the aws xray console

### CODE REVIEW

File: receiver/awsxrayreceiver/internal/translator/translator.go [hunk 1]
```diff
@@ -178,6 +178,8 @@ func populateSpan(
 
 	if parentIDBytes != nil {
 		span.SetParentSpanID(pdata.NewSpanID(parentIDBytes))
+	} else {
+		span.SetKind(2)
```
Comment (anuraaga) [inline receiver/awsxrayreceiver/internal/translator/translator.go:182]:
```suggestion
		span.SetKind(pdata.SpanKindSERVER)
```

**Comment by JohnWu20** [comment]
It seems that reason the build-publish failed is not caused by my code change. Would anyone please help me review the code? @anuraaga

**Review by anuraaga — APPROVED** [review]
For any other reviewers, X-Ray SDK only has two types segments and subsegments. Segments always refer to server spans in X-Ray so the root should be marked as such as this fixed.

Thanks @JohnWu20!

**Comment by tigrannajaryan** [comment]
@anuraaga when the build fails please check CircleCI. If it failed because of a test unrelated to the change then file a bug with the test details (check to see if there is one already existing to avoid duplicates - this one for example already existed, so I just added a link to the failure again: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1093). Then re-run the build so that the PR can be merged.

We (approvers) need to do this since only approvers can rerun CircleCI builds. Extra points for picking up the bug and fixing it :-)

**Comment by JohnWu20** [comment]
@tigrannajaryan Thanks for re-running the test, and it seems that it has passed the CircleCI. Would you please help me merge the code?

**Pull request merged**